lib/bosh/BoshDirectorClient.js

```diff
@@ -358,33 +460,43 @@ class BoshDirectorClient extends HttpClient {
     const query = _.assign({
       limit: 1000
     }, options);
-    return this
-      .request({
-        method: 'GET',
-        url: '/tasks',
-        qs: _.pick(query, ['limit', 'state', 'deployment'])
-      }, 200)
-      .then(res => JSON.parse(res.body));
+
+    return Promise
+      .map(this.activeConfigs, directorConfig => this
```

Comment:
change this.activConfigs to this.activePrimary. We dont need to bother about tasks that are happening on Landscape Bosh.